### PR TITLE
Grammatical Changes and Some minor additions.

### PR DESCRIPTION
**Pull request by ComputerLee**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/computer-science/notation/asymptotic-notation/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: Asymptotic Notation
 
 How do we measure the performance value of algorithms?
 
-Consider how time is one of our most valuable resources. In computing, we can measure performance with the amount of time a process takes to complete. If the data processed by two algorithms is the same, we can decide on the best implementation to solve a problem.
+In Compputer Science, time is one of our most valuable resources. In computing, we can measure performance with the amount of time a process takes to complete. Anthough the difference between the processing time of two algporithms might be extremely small, if the algorithm operates in a loop or has to be executed on a large set of data points, the small difference can become quite significant. If the data processed by two algorithms is the same, we can decide on the best implementation to solve a problem.
```
Comment (co16353sidak) [inline guide/english/computer-science/notation/asymptotic-notation/index.md]: 1. `Compputer Science` -> `Computer Science`
2. `Anthough` -> `Although`
3. `algporithms` -> `algorithms`

**Review by theNaut1lus — CHANGES_REQUESTED** [review]
@ComputerLee 
Small typographical changes in the comments. please consider them

**Comment by thecodingaviator** [comment]
@ComputerLee please make the changes @co16353sidak suggested

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
.

**Review by thecodingaviator — COMMENTED** [review]
Hi,

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Pull request closed without merging**